src/string-bytes.c

```diff
@@ -125,3 +125,64 @@ UINT16 string_bytes_copy_tpm2b(TPM2B *dest, TPM2B *src) {
 
     return rval;
 }
+
+bool string_bytes_is_host_big_endian(void) {
+
+    uint32_t test_word;
+    uint8_t *test_byte;
+
+    test_word = 0xFF000000;
+    test_byte = (uint8_t *) (&test_word);
+
+    return test_byte[0] == 0xFF;
+}
+
```

Comment:
maybe you shall implement string_bytes_endian_convert_x via Micros like what you did for files_read/write_x.